edition/voxel_tool_terrain.h

```diff
@@ -53,6 +56,7 @@ class VoxelToolTerrain : public VoxelTool {
 	static void _bind_methods();
 
 	VoxelTerrain *_terrain = nullptr;
+	float _strength = 1.f;
```

Comment:
I noticed you are using `float` here, but `real_t` in DoSphere. We should probably be consistent. In this case there is no strong reason to prefer either `float` or `real_t`, either works.

The difference is that `real_t` becomes `double` when the engine is compiled with doubles.

Reply:
right. It's always using real